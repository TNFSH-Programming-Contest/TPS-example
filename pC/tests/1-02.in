627395928406.812527 488404210140.920147 804842051805.106822
