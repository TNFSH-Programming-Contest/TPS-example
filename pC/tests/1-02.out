380724158450.60626221
